### PR TITLE
Add pyconsomalia

### PR DESCRIPTION
**Pull request by sidowxyz**

### CODE REVIEW

**Comment by malemburg** [comment]
Hi @ewdurbin, this PR is fine and can be merged. Thank you.

File: _national/pycon-somalia.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: PyCon Somalia
+flag: so
+location: Somalia
+website: http://pycon.org.so/
```
Comment (malemburg) [inline _national/pycon-somalia.yml]: You may want to change this to https instead of http.

Comment (sidowxyz) [reply _national/pycon-somalia.yml]: > You may want to change this to https instead of http.

Thank you sir, i chaneged

**Comment by malemburg** [comment]
Thanks, @ewdurbin

**Pull request merged**